drizzlepac/astrodrizzle.help

```diff
@@ -1009,33 +1008,28 @@ ignore pixel flags of 64 and 32 in the DQ array of the (``_flt.fits``)
 images.  Align the final product such that North is
 up, and set the final pixel scale to 0.05 arcseconds/pixel.
 
-1. Run the task from ``PyRAF`` using the ``TEAL`` GUI::
-
-   >>> import drizzlepac
-   >>> epar astrodrizzle
-
-2. Run the task from ``PyRAF`` using the command line. ::
+1. Run the task from python using the command line. ::
 
     >>> import drizzlepac
     >>> from drizzlepac import astrodrizzle
     >>> astrodrizzle.AstroDrizzle('*flt.fits', output='final',
     ...     wcskey='A', driz_sep_bits='64,32', final_wcs=True,
     ...     final_scale=0.05, final_rot=0)
 
-   Or, run the same task from the PyRAF command line, but specify all
+   Or, run the same task from the python command line, but specify all
    parameters in a config file named ``myparam.cfg``:
 
     >>> astrodrizzle.AstroDrizzle('*flt.fits', configobj='myparam.cfg')
 
-3. Run the task directly from Python:
+2. Run the task directly from Python:
 
     >>> from drizzlepac import astrodrizzle
     >>> astrodrizzle.AstroDrizzle('*flt.fits', output='final',
     ...     wcskey='A', driz_sep_bits='64,32', final_wcs=True,
     ...     final_scale=0.05, final_rot=0)
 
-4. Help can be accessed via the "Help" pulldown menu in the ``TEAL`` GUI.
-   It can also be accessed from the ``PyRAF`` command-line and saved
+3. Help can be accessed via the "Help" pulldown menu in the ``TEAL`` GUI.
+   It can also be accessed from the python command-line and saved
```

Comment:
Did you check this help invocation still works correctly?  The "page" command does not work for me in the basic Python shell.

Reply:
It doesn't for me either. That line has been removed.